src/Microsoft.Health.Fhir.Azure/IntegrationDataStore/AzureAccessTokenClientInitializerV2.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.Storage.Auth;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+
+namespace Microsoft.Health.Fhir.Azure.IntegrationDataStore
+{
+    public class AzureAccessTokenClientInitializerV2 : IIntegrationDataStoreClientInitilizer<CloudBlobClient>
```

Comment:
>AzureAccessTokenClientInitializerV2 [](http://example.com/codeflow?start=17&length=35)

What is the difference between AzureAccessTokenClientInitializer and AzureAccessTokenClientInitializerV2 these look pretty similar?

Reply:
The only difference is AzureAccessTokenClientInitializer requires export configuration as input, we create v2 is a migration from export only to both export and import. To avoid breaking change, first we create V2 version, after new features ready in OSS and PaaS, then we can migrate to AzureAccessTokenClientInitializerV2.